packages/expo-camera/ios/EXCamera/EXCamera.m

```diff
@@ -525,6 +525,11 @@ - (void)record:(NSDictionary *)options resolve:(UMPromiseResolveBlock)resolve re
     }
     [connection setVideoOrientation:[EXCameraUtils videoOrientationForDeviceOrientation:[[UIDevice currentDevice] orientation]]];
 
+    bool shouldBeMirrored = options[@"mirrorVideo"] && [options[@"mirrorVideo"] boolValue];
+    if (shouldBeMirrored) {
```

Comment:
looks like this can never been set as false, have you tested it?
```suggestion
    if (options[@"mirrorVideo"] != nil) {
    
    }
```

Reply:
Nice catch @EvanBacon 👏

Reply:
@EvanBacon the default case is that setVideoMirrored will be false, so not setting the prop will cause this to not be changed.  I've tested that when setting the prop to true, setVideoMirrored is properly set, and when it is left off it is not. I'm happy to change it to be !=nil if you think that is more specific by allowing it to be explicitly set to false.